iStore/Cart/CheckOutButton.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  CheckOutButton.swift
+//  iStore
+//
+//  Created by Mohamed Sayed on 09.03.24.
+//
+
+import SwiftUI
+
+struct CheckOutButton: View {
+    
+    let total: Int
+    let action: () -> Void
```

Comment:
same as above

Reply:
because if private it won't be accessible from the cartView.

Reply:
you can make it private with public init